sensorprocess/sensorprocess_test.go

```diff
@@ -243,6 +243,41 @@ func TestAddSensorReadingLiveReadings(t *testing.T) {
 	})
 }
 
+func invalidSensorTestHelper(
```

Comment:
Can we please also add the following tests for Start:
1. When the context is cancelled before the error is hit
2. When addSensorData returns false

Reply:
added!